app/scripts/models/notifications.js

```diff
@@ -15,7 +15,9 @@ define([
   var EVENTS = {
     DELETE: 'fxaccounts:delete',
     LOGOUT: 'fxaccounts:logout',
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    SIGNED_IN: 'fxaccounts:signedIn',
+    SIGNED_OUT: 'fxaccounts:signedOut'
```

Comment:
Is there a difference between `LOGOUT` and `SIGNED_OUT`? Can we standardize on one alias, and perhaps keep 'fxaccounts:logout' as the actual message that is passed - these messages are sent to the browser and we don't want to cause issues there.